wvd-templates/wvd-management-ux/deploy/scripts/updateWvdMgmtUxApiUrl.ps1

```diff
@@ -14,11 +14,11 @@ Administrator
  Required
  Provide name of the web application which you have used to deploy webapp through ARM Template.
 
-.PARAMETER SubscriptionId
```

Comment:
Revert all "AzureSubscriptionId" to "SubscriptionId" since that's what's available today.